plugins/home/README.md

```diff
@@ -0,0 +1,106 @@
+# Home
+
+Welcome to the home plugin!
```

Comment:
> Welcome to the home plugin!

> _This plugin was created through the Backstage CLI_

These two lines can probably be deleted now as you have been writing good overview documentation.

Reply:
Fixed!